docs/docs/100-reference/01-command-line/acorn_install.md

```diff
@@ -27,6 +27,7 @@ acorn install
       --allow-user-metadata-namespace strings           Allow these namespaces to propagate labels and annotations to dependent objects, no effect if --ignore-user-labels-and-annotations not true
       --api-server-cpu string                           The CPU to allocate to the runtime-api-server in the format of <req>:<limit> (example 200m:1000m)
       --api-server-memory string                        The memory to allocate to the runtime-api-server in the format of <req>:<limit> (example 256Mi:1Gi)
+      --api-server-pod-annotations stringArray          annotations to apply to acorn-api pods
```

Comment:
Is there any chance that we'll need this for the controller pod as well soon?

Reply:
Maybe -- I think we should add it when we need it though.